src/darwin/frida-helper-backend-glue.m

```diff
@@ -1034,6 +1043,100 @@ static void frida_darwin_helper_backend_launch_using_sbs (NSString * identifier,
   }
 }
 
+static void
+frida_darwin_helper_backend_launch_using_lsaw (NSString * identifier, NSURL * url, FridaHostSpawnOptions * spawn_options,
+    FridaDarwinHelperBackendLaunchCompletionHandler on_complete, void * on_complete_target)
```

Comment:
This function should be moved after `frida_darwin_helper_backend_launch_using_sbs` so the definition ordering matches the declaration ordering.